sw/airborne/firmwares/rotorcraft/autopilot_firmware.c

```diff
@@ -128,9 +129,9 @@ static void send_fp(struct transport_tx *trans, struct link_device *dev)
   struct Int32Eulers att = *stateGetNedToBodyEulers_i();
 #endif
   pprz_msg_send_ROTORCRAFT_FP(trans, dev, AC_ID,
-                              &(stateGetPositionEnu_i()->x),
-                              &(stateGetPositionEnu_i()->y),
-                              &(stateGetPositionEnu_i()->z),
+                              &pos->x,
```

Comment:
If our navigation is in floats does it still make sense to send it down in integers? (Maybe something for later to update)

Reply:
yes, indeed, but it is more related to redefining most messages, so it is an other job